File: Famoser.MassPass.Api/src/Models/Response/RefreshResponse.php
<?php
/**
 * Created by PhpStorm.
 * User: famoser
 * Date: 23.05.2016
 * Time: 09:51
 */

namespace Famoser\MassPass\Models\Response;


use Famoser\MassPass\Models\Response\Base\ApiResponse;

class RefreshResponse extends ApiResponse
{
    /*
     * @var Entities\RefreshEntity[]
     */
    public $RefreshEntities;
}

File: Famoser.MassPass.Data/Entities/Communications/Response/RefreshResponse.cs
using System.Collections.Generic;
using Famoser.MassPass.Data.Entities.Communications.Response.Base;
using Famoser.MassPass.Data.Entities.Communications.Response.Entitites;

namespace Famoser.MassPass.Data.Entities.Communications.Response
{
    public class RefreshResponse : ApiResponse
    {
        public RefreshResponse()
        {
            RefreshEntities = new List<RefreshEntity>();
        }

        public List<RefreshEntity> RefreshEntities { get; set; }
    }
}
